.github/workflows/release_android.yml

```diff
@@ -0,0 +1,91 @@
+name: Release Android APK
+
+on: workflow_dispatch
+
+env:
+  FLUTTER_VERSION: "3.3.1"
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # #499, https://github.com/actions/virtual-environments/issues/5595
+      - name: Configure ndk
+        run: |
+          ANDROID_HOME=$HOME/Library/Android/sdk
+          SDKMANAGER=$ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager
+
+          echo y | $SDKMANAGER "ndk;21.4.7075529"
+
+          ln -sfn $ANDROID_HOME/ndk/21.4.7075529 $ANDROID_HOME/ndk-bundle
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: "temurin"
+          java-version: "8.x" # "betterprogramming.pub" says must be java "8"
+          cache: "gradle"
+
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: "stable"
+          flutter-version: "3.3.1"
```

Comment:
Can you use the env variable from the top here so that we don't have two versions in this file?